capstone/src/main/java/com/google/step/servlets/PageViewServlet.java

```diff
@@ -0,0 +1,132 @@
+package com.google.step.servlets;
+
+import com.google.gson.Gson;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+import java.util.List;
+
+import com.google.step.clients.MetricsClient;
+import com.google.step.data.WeeklyPageView;
+import com.google.step.data.RestaurantPageViews;
+
+/**
+ * Servlet that will call the metricsClient for any doGet or doPost requests.
+ */
+@WebServlet("/page-view")
+public class PageViewServlet extends HttpServlet {
+
+    private static MetricsClient metricsClient = new MetricsClient();
+
+    /**
+     *
+     * @param request with a mix of the following parameters: <String, restaurantKey> and <int, year>.
```

Comment:
Nit: Keep in mind that the formatted javadoc comment won't show anything in angle brackets.

Reply:
What do you mean the formatted javadoc?

Reply:
Anyone who doesn't necessarily have access to the code is only going to have the method declaration and the javadocs to look at when using your code. If I'm not mistaken, angle brackets in javadocs are interpreted as HTML tags. So `<String, restaurantKey>` wouldn't show up in a formatted javadoc that one might be viewing if they were looking at documentation for the code.

Reply:
It's not a super big deal, but I think it's good to know and keep in mind.

Reply:
Ah alright, will change that for the future.